### PR TITLE
Flutter Dockerfile: use apt-get, combine tasks

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: .cloud_build/flutter/Dockerfile [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
-  libglu1-mesa
+RUN apt-get update -y  \
+    && apt-get install -y \
+    git \
+    wget \
+    curl \
+    unzip \
+    lib32stdc++6 \
+    libglu1-mesa \
+    && rm -rf /var/lib/apt/lists/*
```
Comment (johnpryan) [inline .cloud_build/flutter/Dockerfile:15]: nit: Should we run `apt-get clean` to delete package files also? I assume we're trying to reduce the container size?

**Review by johnpryan — APPROVED** [review]

File: .cloud_build/flutter/Dockerfile (hunk 1 above)
Comment (kevmoo) [reply .cloud_build/flutter/Dockerfile:15]: I'm just going off what gemini told me 😄 

Let me look...

Comment (kevmoo) [reply .cloud_build/flutter/Dockerfile:15]: https://docs.docker.com/build/building/best-practices/#sort-multi-line-arguments

Comment (kevmoo) [reply .cloud_build/flutter/Dockerfile]: Found more info. both is good!

Comment (johnpryan) [reply .cloud_build/flutter/Dockerfile]: I guess I should just ask Gemini next time! Thanks for digging in.

**Pull request merged**